Framework/DataHandling/src/CompressEvents.cpp

```diff
@@ -37,22 +40,51 @@ void CompressEvents::init() {
       "The tolerance on each event's X value (normally TOF, but may be a "
       "different unit if you have used ConvertUnits).\n"
       "Any events within Tolerance will be summed into a single event.");
+
+  declareProperty(
+      make_unique<PropertyWithValue<double>>("WallClockTolerance", EMPTY_DBL(),
+                                             mustBePositive, Direction::Input),
+      "The tolerance on the wall-clock time for comparison. Unset means "
+      "compressing all wall-clock times together.");
+
+  auto dateValidator = boost::make_shared<DateTimeValidator>();
+  dateValidator->allowEmpty(true);
+  declareProperty(
+      "StartTime", "", dateValidator,
+      "An ISO formatted date/time string specifying the timestamp for "
+      "starting filtering (e.g 2010-09-14T04:20:12). Ignored if "
```

Comment:
The doc string will be cut at the first dot, between `e.g`, for the tooltip in the automatically generated dialog. The string should therefore be rephrased to drop the in-sentence dot.